x/operators/keeper/genesis.go

```diff
@@ -65,15 +67,14 @@ func (k *Keeper) InitGenesis(ctx sdk.Context, state *types.GenesisState) error {
 	// Store the operator params
 	for _, operatorParams := range state.OperatorsParams {
 		// Ensure that the operator is present
-		_, found, err := k.GetOperator(ctx, operatorParams.OperatorID)
+		_, err := k.GetOperator(ctx, operatorParams.OperatorID)
 		if err != nil {
+			if errors.Is(err, collections.ErrNotFound) {
+				return fmt.Errorf("can't set operator params for %d, operator not found", operatorParams.OperatorID)
```

Comment:
The same here: let's use `types.ErrOperatorNotFound` instead. We can always wrap that for a customized error message, but I would use typed errors instead of new ones